### PR TITLE
Redirect if no valid token

### LINKED ISSUE #141: Unauthenticated error should redirect back to login page
If the token is no longer valid, any requests to the /graphql endpoint will result in a 400 error.

Currently the client just shows a blank screen. It should redirect to the login screen

### PR DESCRIPTION
**Pull request by norisalsaadie**
Closes #141 

We need to handle this in a better way, but it makes sense to do so when generic error handler is considered.

### CODE REVIEW

File: src/core/utils/createApolloClient.ts [hunk 1]
```diff
@@ -24,5 +24,21 @@ export default (host: string): ApolloClient<unknown> => {
         request: async (operation): Promise<void> => {
             await operation.setContext(getHeaders(getToken()));
         },
+        // TODO: figure out generic handler for errors
+        onError: ({ graphQLErrors = [], networkError }): void => {
+            graphQLErrors.forEach(({ message, locations, path }) =>
+                console.log(`[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`),
```
Comment (Cooryd) [inline src/core/utils/createApolloClient.ts]: Should we be `console.log`ing in here?

Comment (norisalsaadie) [reply src/core/utils/createApolloClient.ts]: For now, yes. See the todo on line 27. i think we need to sort out error handling properly.

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "license": "MIT",
     "scripts": {
         "build": "webpack --env production",
-        "start": "webpack-dev-server --env development",
+        "start": "PUBLIC_CONFIG_PATH=config/config.public.json INFRA_CONFIG_PATH=config/config.infra.json webpack-dev-server --env development",
```
Comment (erezmus) [inline package.json]: why have the env vars been re-added? its in the make file

Comment (norisalsaadie) [reply package.json]: Good catch. It is new branch because the yarn lock went wild. Addressing.

**Pull request merged**